cascade-api/src/Cascade/Api/Network/Server/Api/Authentication.hs

```diff
@@ -41,10 +42,10 @@ import           Validation                          ( validation )
 import           Web.Cookie
 import qualified Web.Cookie                         as Cookie
 
-handleLogin :: Members '[Database.UserL , Error ServerError] r => Authentication.RawCredential -> Sem r LoginResponse
-handleLogin = validation (const $ throw err422) go . parseRawCredential
+handleLogin :: Members '[Database.UserL , Error ServerError] r => Authentication.Credential 'Validation.Raw -> Sem r LoginResponse
+handleLogin = validation (const $ throw err422) go . Authentication.parseRawCredential
```

Comment:
Login doesn't respond with validation errors, is that intentional?

Reply:
It was. Actually, it was ignored by #17.
BTW, I think it's better to report validation errors.